.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build:
+  release:
```

Comment:
We will also need to change the "required status check context" in `infrastructure` to match this new name: https://github.com/fiberplane/infrastructure/blob/main/github/repositories.tf#L126-L134

Reply:
I don't think this one should be blocking. There's still a `build` job left, which is the one that tests PRs. I renamed this one because there were two `build` jobs, which was rather confusing.

Reply:
ah good point :+1: